k8s-install/scripts/install-cni.sh

```diff
@@ -128,18 +128,23 @@ sed -i s~__ETCD_CA_CERT_FILE__~${CNI_CONF_ETCD_CA:-}~g $TMP_CONF
 sed -i s~__ETCD_ENDPOINTS__~${ETCD_ENDPOINTS:-}~g $TMP_CONF
 sed -i s~__LOG_LEVEL__~${LOG_LEVEL:-warn}~g $TMP_CONF
 
-FILENAME=${CNI_CONF_NAME:-10-calico.conf}
+CNI_CONF_NAME=${CNI_CONF_NAME:-10-calico.conf}
+CNI_OLD_NAME=${CNI_OLD_NAME:-10-calico.conf}
```

Comment:
One minor thing - I think it would be good to name the config option `CNI_OLD_CONF_NAME` to make it more clear it's referring to the old CNI configuration file name, not something else.

Reply:
sure thing!